src/js/hooks/list-pages/outbound/useOutboundListTableData.jsx

```diff
@@ -132,6 +133,34 @@ const useOutboundListTableData = (filterParams) => {
     });
   };
 
+  const approveRequest = async (id, identifier) => {
+    dispatch(showSpinner());
+    try {
+      await stockMovementApi.updateStatus(id, RequisitionStatus.APPROVED);
+      const successMessage = `You have successfully Approved the request ${identifier}`;
+      Alert.success(successMessage);
+      fireFetchData();
+    } finally {
+      dispatch(hideSpinner());
+    }
+  };
+
+  const rejectRequest = async (id, identifier) => {
+    dispatch(showSpinner());
+    try {
+      await stockMovementApi.updateStatus(id, RequisitionStatus.REJECTED);
+      const successMessage = `You have successfully Rejected the request ${identifier}`;
+      Alert.success(successMessage);
+      fireFetchData();
+    } finally {
+      dispatch(hideSpinner());
+    }
+  };
+
+  const rollbackRequest = () => {
+    Alert.error('Not implemented');
```

Comment:
will it be handled in another ticket or are you waiting for it to be merged?

Reply:
I am waiting for the changes to be merged.
I can either wait for it or add it in the following PR.